RocketChat/Rocket.Chat.ReactNative
# [RKT-17] Animated GIFs from Photos on iOS upload as JPGs and are static. #4178

## Proposed changes

1- GIF images was forced changed to JPG, set bool to `forceJpg: false`, also library 'react-native-image-crop-picker' was saving .gif files as .jpg in tmp directory so instead of using tmp directory url for uploading .gif now using original sourcePath.
2- List was not displaying .gif and audios uploaded from iOS properly it was showing these as Reply so check the issue and some how iOS uploaded audio and gif files do not having `audio_url` and `image_url` so check for `title_url` and fixed issue.
## Issue(s)	

[#4178](https://github.com/RocketChat/Rocket.Chat.ReactNative/issues/4178)

## How to test or reproduce


1- Open App on iOS device, click on + sign inside a Room and send a .gif image.
2- Image will appear static instead of animated.

## Screenshots

## Types of changes




## Checklist



## Further comments


Bug video: https://www.loom.com/share/4998f29fbde74e388e480501bc9fa8d4
After Fix Video: https://www.loom.com/share/fbfe787430ba4f91afe8992098544253
